src/PointwiseFunctions/Hydro/EquationsOfState/Python/Enthalpy.cpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Hydro/EquationsOfState/Python/Enthalpy.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.hpp"
+
+// low_density_eos uses Spectral and Polytropic Fluid type EoS.
+#include <pybind11/numpy.h>
+#include <pybind11/operators.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <string>
+#include <vector>
+#include "PointwiseFunctions/Hydro/EquationsOfState/PolytropicFluid.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/Spectral.hpp"
+
+namespace py = pybind11;
+
+namespace EquationsOfState::py_bindings {
+
+void bind_enthalpy(py::module& m) {
+  py::class_<Enthalpy<PolytropicFluid<true>>, EquationOfState<true, 1>>(
```

Comment:
Please combine these four functions into one. To do this, create a new function template `bind_enthalpy_impl<LowDensityEos>` that you call four times with different template parameters. You can pass the name of the class as a string to the function.